owm_legacy/__init__.py

```diff
@@ -1,4 +1,4 @@
-VERSION = (0, 4, 1, 'final')
+VERSION = (0, 5, 1, 'final')
```

Comment:
do not edit this in this commit, I will create a new to tag it the change of version (0.5.0, not 0.5.1).